### PR TITLE
Created Solution for "CSES - 1740"

### PR DESCRIPTION
**Pull request by dutinmeow**

### CODE REVIEW

**Comment by dutinmeow** [comment]
Is Vercel failing normal?

**Comment by maggie-j-liu** [comment]
no, it's not normal 👀 
you can click on "Preview" 
image
to check the error (it tells you how to fix it)

**Comment by dongliu0426** [comment]
this is already [a focus problem](https://usaco.guide/plat/range-sweep?lang=cpp#solution---intersection-points)

File: content/5_Plat/Range_Sweep.problems.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
       "isStarred": false,
       "tags": ["PURS"],
       "solutionMetadata": {
-        "kind": "none"
+        "kind": "internal"
```
Comment (dongliu0426) [inline content/5_Plat/Range_Sweep.problems.json:13]: change this to 
```
      "solutionMetadata": {
        "kind": "in-module",
        "moduleId": "range-sweep"
      }
```

**Review by dongliu0426 — COMMENTED** [review]
also this problem is in plat/sweep-line as well

**Comment by dutinmeow** [comment]
> this is already [a focus problem](https://usaco.guide/plat/range-sweep?lang=cpp#solution---intersection-points)

Maybe just put a link from the RQ with SL to the solution? I use order-statistics tree instead of BIT

**Comment by ar492** [comment]
> Maybe just put a link from the RQ with SL to the solution? I use order-statistics tree instead of BIT

what is rq and sl

**Comment by dongliu0426** [comment]
> > this is already [a focus problem](https://usaco.guide/plat/range-sweep?lang=cpp#solution---intersection-points)
> 
> Maybe just put a link from the RQ with SL to the solution? I use order-statistics tree instead of BIT

sure

**Comment by dongliu0426** [comment]
> > Maybe just put a link from the RQ with SL to the solution? I use order-statistics tree instead of BIT
> 
> what is rq and sl

range queries, sweep line

**Comment by dutinmeow** [comment]
It still says that I'm missing the solution link from one of the modules

**Comment by dongliu0426** [comment]
> also this problem is in plat/sweep-line as well

@dutinmeow

**Comment by dutinmeow** [comment]
It should be fixed now, I thought that range sweep and sweep line were the same module for some reason

**Comment by dutinmeow** [comment]
Vercel is dumb

**Comment by dutinmeow** [comment]
How do I fix the issue now -_-

**Comment by dutinmeow** [comment]
Already has solution

**Pull request closed without merging**